### DISCUSSION TITLE
MMPBSA——index error: list index out of range

### DISCUSSION THREAD

**Discussion by sf-polaris** [Valdes-Tresanco-MS/gmx_MMPBSA#444] [Q&A]
![image](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/1440be33-e481-4321-8330-a4979f4b348c)

Hi，I want to calculate the binding free energy of the membrane  protein . But I cannot submit the task successfully.

**Comment by sf-polaris** [comment]
This is my Topol files
![image](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/970e2eeb-2fb7-42e3-bf5d-82d7a18b7465)
![image](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/ce003b59-cb55-4c3a-856a-d4a98a6758c7)
And my " xtc " file only contain protein and ligand.

**Comment by Valdes-Tresanco-MS** [comment]
Please, check that any defined groups don't contain ions

**Reply by sf-polaris** [reply]
Yes, my topol files and xtc files don't contain ions, I use the CHARMM fileds.

**Reply by Valdes-Tresanco-MS** [reply]
Please, send me your files to check them

**Reply by sf-polaris** [reply]
[Desktop.zip](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/13734453/Desktop.zip)
OK, Here are my whole files. Thank you so much!!! It has  been bothering me for a long time.

**Reply by sf-polaris** [reply]
And I use GROMACS 2018.8 , CHARMM fields 2019, gmx_MMPBSA 1.3.3

**Comment by Valdes-Tresanco-MS** [comment]
gmx_MMPBSA 1.3.3? This version is old, so it is not maintained anymore. Please, try the latest version

**Reply by sf-polaris** [reply]
<img width="398" alt="屏幕截图 2023-12-21 110614" src="https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/8249b66a-57e2-4e36-9c02-51cfebbd9b63">
It may be some problems of our machine.

**Reply by sf-polaris** [reply]
We installed the software not long ago. The dependent sofware installed are not old. But it is weird the gmxMMPBSA version is so old.

**Comment by sf-polaris** [comment]
I also have a question. I have been successfully submitted the task a month ago. I checked the decomposition files and found the non- polar items are zero. Is it correct?
 
<img width="926" alt="1703141467333" src="https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/134beec5-c1fe-48fd-921c-2d436205f94e">
